### DISCUSSION TITLE
Tasklist beta access / Roadmap

### DISCUSSION THREAD

**Discussion by dkozma** [community/community#72709] [Projects and Issues]
### Select Topic Area

Question

### Feature Area

Issues

### Body

Our team is currently evaluating managing our open source projects workflow on GitHub Projects, and tasklists (v2) are the one thing that we need to actually make this a viable option. I realize it is still in private beta, however it looks like onboarding is closed. I also have searched the roadmap and don't see any mention about a public release. All I see is an issue for hierarchal tasklists, which is more functionality than we currently need (we just require an epic/tracking issue solution to group related issues together).

Is there a timeframe as to when new people will be granted access to the beta or a public release will happen? It would be great to be in a position to be able to provide feedback, and it would be a shame to have to find another solution for this that doesn't integrate as cleanly into GitHub.

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by LBWright** [comment]
This is something we need, as well. We would like to use this for tracking epics and milestones more cleanly in our project.
